src/Kanbas/Courses/Quizzes/index.tsx

```diff
@@ -1,3 +1,12 @@
-export default function Quizzes() {
-  return <div>test</div>;
-}
+import { Link } from "react-router-dom";
+
+function Quizzes() {
+  return (
+    <div className="flex-fill d-flex gap-5">
+      <Link to={`./1`}>Editor</Link>

```

Comment:
should we leave a todo here to remember to update this to the correct quiz id?

Reply:
yep I can add that in